static/js/room.js

```diff
@@ -645,16 +645,16 @@ window.addEventListener('load', () => {
 
     // Buttons for preset themes.
     document.getElementById('light-theme-button').onclick = () => changeColors({
```

Comment:
Instead of changing the colors here, you should copy lines 647-663 for your button and change `light-theme-button` to `cool-theme-button`.